internal/re.js

```diff
@@ -159,7 +159,15 @@ createToken('COERCE', `${'(^|[^\\d])' +
               `(?:\\.(\\d{1,${MAX_SAFE_COMPONENT_LENGTH}}))?` +
               `(?:\\.(\\d{1,${MAX_SAFE_COMPONENT_LENGTH}}))?` +
               `(?:$|[^\\d])`)
+createToken('COERCEFULL', `${'(^|[^\\d])' +
```

Comment:
If this is all duplicating `COERCE` and then appending new content, it should reuse `t.COERCE`.  You can look to things like `LOOSEPLAIN` as an example.